docs/quick-start-guide.md

```diff
@@ -89,7 +89,7 @@ spec:
     labels:
       version: 2.3.0
     memory: 512m
-  image: gcr.io/ynli-k8s/spark:v3.0.0
+  image: gcr.io/ynli-k8s/spark:v3.1.1
   mainApplicationFile: local:///opt/spark/examples/jars/spark-examples_2.12-2.3.0.jar
```

Comment:
Shouldn't this be **local:///opt/spark/examples/jars/spark-examples_2.12-3.1.1.jar**

Reply:
Right, updated it. Thanks :)